index.rb

```diff
@@ -126,6 +126,24 @@ def show(**kwargs)
   end
 end
 
+venues << Venue.new(:name => 'DNA Lounge', :link => 'https://www.dnalounge.com/calendar/dnalounge.rss') do
+  # Regex to dig the calendar link out of the description
+  link_regex = Regexp.new(/https:\/\/www.dnalounge.com\/calendar\/\d{4}\/[\d\w\-]+\.html/i)
+
+  URI.open(link) do |xml|
+    Nokogiri::XML(xml).css('item').map do |item|
+      description = item.css('description').text
+
+      show(
+        time: Time.parse(item.css('dnalounge|date').text),
+        link: description[link_regex] || item.css('guid').text,
```

Comment:
Why the regex? Is `guid` not always there? Seems like it's a link to the calendar or flyer.

Reply:
In most cases, guid links to the flier, which isn't ideal